Blogpost regarding Managing R packages community call

steffilazerte opened the pull request:
Here's a basic draft but missing cohesion

* [X] I have read the Content Guidelines.
* [X] I have read the Technical Guidelines.
* [X] I used or followed the R Markdown or Markdown template.
* [X] I have followed the Style Guide.
* [X] I created or updated my author metadata with correct folder name.
* [X] I have added relevant tags after browsing existing tags.
* [X] I ran `roblog::ro_lint_md()` on index.md (optional).
* [X] I ran `roblog::ro_check_urls()` on index.md (optional).

jananiravi commented:
Hello @stefaniebutland @steffilazerte, I have a few notes but I have not been able to edit the post or submit a PR for file. Should I just make notes and convey them during the Friday call? Would that be easier?

stefaniebutland commented:
No worries @jananiravi. Could you put your notes in a google doc and share with Steffi and me before Friday? No worries if bullet points. We can use that doc for more notes when we meet

Would be very helpful if you commented - in the doc or when we meet - on this preview of what Steffi has put together so we can incorporate your feedback if any https://deploy-preview-708--ropensci.netlify.app/blog/2020/05/26/commcall-maintaining-pkg/

Thank you!

steffilazerte commented:
I've made the edits we discussed plus a couple of minor changes:

- @jananiravi I changed the first place where you switched to third person to first person with a clarifier. I think moved the sentence describing how you took notes further down and also switched it to first person. I think this captures your voice without implying that the I is both of us. What do you think? Is this weird or does it work for you?

- @jananiravi @stefaniebutland How do you both feel about the multiple thumbs up emojis at the bottom? I've been reading about inclusivity in emojis and I'd like to try to put some of what I've learned into practice. Any reason not to do what I've suggested? Anything I may not have understood/overlooked?

jananiravi commented:
That sounds good, @steffilazerte 

Sure, the graded one looks good or would just a yellow work (isn't that supposed to be neutral or is that supposed to mean yellow-skinned?)!

steffilazerte commented:
I always thought the yellow meant neutral, but from what I've read, it's compared to the Simpsons characters, which means: Yellow = White.

I think they need a blue emoji for neutral :wink:

jananiravi commented:
OK, then! There’s a :thumbsup_all: on Slack. You could see if that works in GitHub as well. That might solve the problem!

stefaniebutland commented:
Suggestion for title: Maintaining an R Package - A Community Call Summary

Why? put content focus first. Title as is sounds like we're announcing a new call

steffilazerte commented on content/blog/2020-07-14-commcall-maintaining-pkg/index.md:
```diff
@@ -0,0 +1,116 @@
+
+**2. How do you manage issues / feature requests?** ([document](https://docs.google.com/document/d/1KvuVNU4ICE_FHvJMAfKns_pZnOt_1cgJ4wokxGGuDaE/edit#heading=h.89gqj51qdulg))  
+**What workflows do you use to do this?** ([document](https://docs.google.com/document/d/1KvuVNU4ICE_FHvJMAfKns_pZnOt_1cgJ4wokxGGuDaE/edit#heading=h.ccqqertq2gox))  
+([video - All](https://vimeo.com/399048757#t=22m04s))
```
Done!

steffilazerte commented:
Okay, @stefaniebutland all the suggestions made sense so I accepted them all, and I think we're good to go!
